libs/utils/env.py

```diff
@@ -439,6 +441,54 @@ def _init_target(self, force = False):
             self.__modules = ['bl', 'cpufreq']
             platform = Platform(model='pixel')
 
+        # Initialize gem5 platform
+        elif self.conf['board'].upper() == 'GEM5':
+            self.nrg_model = None
+            self.__modules=['cpufreq']
+
+            system_info = self.conf['board_settings']['system']
+            simulator_info = self.conf['board_settings']['simulator']
```

Comment:
The idea was to try to keep the general structure somewhat generic, e.g. use it for several "boards", but I don't know how realistic that is...